class.phpmailer.php

```diff
@@ -1364,20 +1364,24 @@ public function postSend()
      */
     protected function sendmailSend($header, $body)
     {
-        // CVE-2016-10033, CVE-2016-10045: Don't pass -f if characters will be escaped by escapeshellcmd when popen is called due to safe mode.
-        if (!empty($this->Sender) || ini_get('safe_mode')) {
+        // CVE-2016-10033, CVE-2016-10045: Don't pass -f if characters will be escaped.
```

Comment:
I think the cat was out of the bag some years ago: https://bugs.php.net/search.php?cmd=display&search_for=escapeshellarg, but it doesn't look like much has been done.